File: ios/sources/commands/ARESSaveCommand.m
//
//  ARESSaveCommand.m
//  ares
//
//  Created by PonyCui on 2018/11/30.
//  Copyright © 2018年 XT Studio. All rights reserved.
//

#import "ARESSaveCommand.h"
#import "ARESView.h"

@implementation ARESSaveCommand

- (void)draw:(CGContextRef)ctx view:(ARESView *)view {
    [view save];
}

@end

File: android/ares/src/main/java/com/xt/ares/commands/ARESSaveCommand.kt
package com.xt.ares.commands

import android.graphics.Canvas
import com.xt.ares.ARESCommand
import com.xt.ares.ARESView

class ARESSaveCommand: ARESCommand() {

    override fun draw(view: ARESView, canvas: Canvas) {
        super.draw(view, canvas)
        view.save(canvas)
    }

}